src/main.rs

```diff
@@ -183,6 +192,23 @@ pub struct Log {
     pub message: Option<String>,
 }
 
+fn serde_json_simple_test() {
+#[derive(Serialize, Deserialize, Debug)]
+  struct Pnt {
+    x: i32,
+    y: i32,
+  }
+
+  let point      = Pnt { x: 1, y: 2 };
+  let serialized = serde_json::to_string(&point).unwrap();
+  println!("serialized = {}", serialized);
```

Comment:
instead of a `println!` macro, do the `assert_eq!` macro with the string you expec